src/Explorer.js

```diff
@@ -599,6 +618,7 @@ class AbstractArgView extends React.PureComponent<AbstractArgViewProps, {}> {
               setArgValue={this.props.setArgValue}
               arg={arg}
               argValue={argValue}
+              onRunOperation={() => this.props.onRunOperation()}
```

Comment:
use class method

Reply:
Just replaced with `onRunOperation={this.props.onRunOperation}`